ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -314,16 +315,24 @@ hideall "--sync--"
 3535.8 "Staggering Breath" #sync / 1[56]:[^:]*:Nidhogg:6D3D:/
 3543.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D30:/
 3554.3 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3558.3 "--nidhogg untargetable--"
```

Comment:
What do you think about just making these be untargetable/targetable since they don't overlap? There's only a single one going away and back at any time, and it's pretty obvious which it is.

Reply:
That works for me.